Maddori.Apple/Maddori.Apple/Screen/AddFeedbackContent/AddFeedbackContentViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AddFeedbackContentViewController.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class AddFeedbackContentViewController: BaseViewController {
+    
+    // MARK: - property
+
+    private let backButton = BackButton(type: .system)
+    private let exitButton = ExitButton(type: .system)
+    private let addFeedbackTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.addFeedbackContentViewControllerTitleLabel
+        label.textColor = .black100
+        label.font = .title
+        return label
+    }()
+    private let feedbackTypeLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.addFeedbackContentViewControllerFeedbackTypeLabel
+        label.textColor = .black100
+        label.font = .label2
+        return label
+    }()
+    private let feedbackTypeButtonView = FeedbackTypeButtonView()
+    
+    // MARK: - lifecycle
+    
+    override func render() {
+        view.addSubview(addFeedbackTitleLabel)
+        addFeedbackTitleLabel.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide).offset(SizeLiteral.topPadding)
+            $0.leading.equalToSuperview().offset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(feedbackTypeLabel)
+        feedbackTypeLabel.snp.makeConstraints {
+            $0.top.equalTo(addFeedbackTitleLabel.snp.bottom).offset(40)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(feedbackTypeButtonView)
+        feedbackTypeButtonView.snp.makeConstraints {
+            $0.top.equalTo(feedbackTypeLabel.snp.bottom).offset(10)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+    }
+
+    // MARK: - func
+    
+    override func setupNavigationBar() {
+        super.setupNavigationBar()
+        
+        let backButton = makeBarButtonItem(with: backButton)
+        let exitButton = makeBarButtonItem(with: exitButton)
+        
+        navigationController?.navigationBar.prefersLargeTitles = false
+        navigationItem.largeTitleDisplayMode = .never
+        navigationItem.leftBarButtonItem = backButton
+        navigationItem.rightBarButtonItem = exitButton
+    }
```

Comment:
p1; 

```suggestion
    override func setupNavigationBar() {
        super.setupNavigationBar()
        
        let button = removeBarButtonItemOffset(with: backButton, offsetX: 10)
        let backButton = makeBarButtonItem(with: button)
        let exitButton = makeBarButtonItem(with: exitButton)
        
        navigationController?.navigationBar.prefersLargeTitles = false
        navigationItem.largeTitleDisplayMode = .never
        navigationItem.leftBarButtonItem = backButton
        navigationItem.rightBarButtonItem = exitButton
    }
```

요 코드로 고쳐주세요 ! @seongmin221 의 예리한 눈으로 네비바의 왼쪽 버튼의 위치가 다른걸 캐치했습니다 ! 감사합니다 !!
@Guel-git 이미 merge가 가능한 상태지만 요거 꼭 반영해주세요 !

Reply:
헿헿 캐치한건 저지만 고친건 @MMMIIIN  입니다 ㅎ 감사해요 👀

Reply:
근데 문득  든 생각인데 쟤를 아예 네브바 컴포넌트 만든곳에서 enum으로 두고 (enum 성애자...) type 만 지정하면 알아서 네브바에 들어가는 요소들 바꾸게 하면 어떨까요??

Reply:
어머머 대박적 👁매의 눈👁 이드와 ✋빠른 손✋ 케미의 🌀콜라보레이션🌀 ~~
[35f3280](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/35f32809270ceed2e90a9dc156a97e0fa585956c)
수정해서 반영해두었습니다!

Reply:
@seongmin221 굉장히 좋은 방법이네요 ! 저도 enum 성애자입니다...
한번 생각해보겠습니다 ! 감사합니다 !